README.md

```diff
@@ -33,4 +50,4 @@ Union Road,
 Farnham, 
 Surrey, 
 England,
-GU9 7PT
\ No newline at end of file
+GU9 7PT
```

Comment:
lol: no idea. Git is funny sometimes. It has moved down a significant number of lines so maybe git just playing it safe?